src/Sarif.WorkItems/SarifWorkItemFiler.cs

```diff
@@ -238,21 +255,29 @@ private static void UpdateLogWithWorkItemDetails(SarifLog sarifLog, Uri htmlUri,
             }
         }
 
-        private void LogMetricsForFiledWorkItem(SarifLog sarifLog, SarifWorkItemModel sarifWorkItemModel)
+        private void LogMetricsForFiledWorkItem(SarifLog sarifLog, SarifWorkItemModel sarifWorkItemModel, FilingResult filingResult)
         {
-            string guid = Guid.NewGuid().ToString();
+            this.FilingResult = filingResult;
+
+            string workItemGuid = Guid.NewGuid().ToString();
```

Comment:
>workItemGuid [](start = 19, length = 12)

If the logGuid is generated in FileWorkItemsHelper() and one work item is filed per split sarif log, what's the difference between workItemGuid and logGuid? It seems like there will always be a 1:1 correlation between them.